packages/components/src/Link/docs/size.tsx

```diff
@@ -0,0 +1,17 @@
+import { Link, Stack, Text } from "@hopper-ui/components";
+
+export default function Example() {
+    return (
+        <Stack>
+            <Text size="xs">
+                Would you like to <Link size="inherit" href="#">learn more</Link> about this fine component?
```

Comment:
Would you like to <Link size="inherit" href="#">learn more</Link> about this role would fit better